Paranormal/Paranormal/ToolSettings.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+public class ToolSettings {
+    weak var document : Document?
+
+    init(){
+    }
+
+    public var colorAsAngles : Array<Float> = [0,0] {
+        didSet {
+            updateToolSettingsView()
+        }
+    }
+
+    public func updateToolSettingsView() {
+        let panelVC = document?.singleWindowController?.panelsViewController
+        panelVC?.toolSettingsViewController?.updateSettingsSliders()
```

Comment:
This seems a little sketchy and global. That being said, I hate this stuff and not sure what I would have done. Maybe a notification? I would say leave it as it works for now but just makes me sad :p. @kotoole1 what are your thoughts on stuff like this?